### DISCUSSION TITLE
Is JAVA API only way to update a graph property?

### DISCUSSION THREAD

**Discussion by dianaoa22** [ArcadeData/arcadedb#1588] [General]
Test is a vertex with property 'admin_state' to be set to 'disable'

MATCH {type: Test, as: t, where: (ipaddress = '18.235.150.109')} RETURN t.admin_state;
+--------------+------+
|NAME          |VALUE |
+--------------+------+
|t.admin_state|enable|
+--------------+------+
Command executed in 1090ms

However not able to update this property from SQL/Cypher.

{nedata}> UPDATE Test SET admin_state = 'disable';

ERROR:
com.arcadedb.exception.TransactionException: Transaction not active

{nedata}> MATCH {type: Test as: ne, where: (ipaddress = '18.235.150.109')} SET ne.admin_state ='disable' RETURN ne;

ERROR:
com.arcadedb.exception.CommandSQLParsingException: MATCH {type: Test, as: ne, where: (ipaddress = '18.235.150.109') SET ne.admin_state ='disable' RETURN ne

**Comment by lvca** [comment]
Are you using the console as embedded or with a remote connection?

The error about the transaction is because you can't update the database outside a transaction. Try doing this:

```
begin;
UPDATE Test SET admin_state = 'disable';
commit;
```

The console should have the auto transaction = true by default. It could be a bug. Please let me know if works with explicit transaction.

**Reply by dianaoa22** [reply]
I am using console as remote, Thanks the above suggestion worked.
